openpgp/write.go

```diff
@@ -275,14 +279,28 @@ func writeAndSign(payload io.WriteCloser, candidateHashes []uint8, signed *Entit
 		return nil, errors.InvalidArgumentError("cannot encrypt because no candidate hash functions are compiled in. (Wanted " + name + " in this case.)")
 	}
 
+	var salt []byte
 	if signer != nil {
+		var sigVersion = 3
```

Comment:
Nit: this might imply the signature itself is also v3. How about
```suggestion
		var opsVersion = 3
```